Backend/model/model.py

```diff
@@ -16,6 +16,13 @@ class Ngo(BaseModel):
     location: str
     pincode: int
 
+
+class Donor(BaseModel):
```

Comment:
make this class down side, and let the validator inside ngo class

Reply:
done